bundles/org.openhab.binding.buienradar/src/main/java/org/openhab/binding/buienradar/internal/BuienradarHandler.java

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.buienradar.internal;
+
+import static java.util.concurrent.TimeUnit.MINUTES;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.time.ZonedDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.concurrent.ScheduledFuture;
+
+import javax.measure.Unit;
+import javax.measure.quantity.Speed;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.library.types.QuantityType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.thing.ThingStatus;
+import org.eclipse.smarthome.core.thing.ThingStatusDetail;
+import org.eclipse.smarthome.core.thing.binding.BaseThingHandler;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.buienradar.internal.buienradarapi.BuienradarPredictionAPI;
+import org.openhab.binding.buienradar.internal.buienradarapi.Prediction;
+import org.openhab.binding.buienradar.internal.buienradarapi.PredictionAPI;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import tec.uom.se.unit.Units;
+
+/**
+ * The {@link BuienradarHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Edwin de Jong - Initial contribution
+ */
+@NonNullByDefault
+public class BuienradarHandler extends BaseThingHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(BuienradarHandler.class);
+
+    public static final Unit<Speed> MM_PER_HOUR = Units.METRE.divide(1000).divide(Units.HOUR).asType(Speed.class);
+
+    @Nullable
+    private BuienradarConfiguration config;
```

Comment:
Please use the pattern `private @Nullable ..` for the nullable annotation

Reply:
Done